simtools/layout/layout_array_builder.py

```diff
@@ -0,0 +1,244 @@
+import astropy.units as u
+import matplotlib.patches as mpatches
+import numpy as np
+from astropy.table import QTable
+from matplotlib import pyplot as plt
+from matplotlib.collections import PatchCollection
+
+from simtools import io_handler
+from simtools.util import names
+from simtools.visualization import legend_handlers as leg_h
+
+
+class LayoutArrayBuilder:
+    """
+    Manage the array layout. It provides methods to create and plot a LayoutArray.
```

Comment:
These are two very different tasks in one class (create a layout and plot it). Would it make sense to separate that?

Reply:
Yes, indeed I moved now plot-related stuff to visualization/visualize.py